lib/registration.pm

```diff
@@ -155,17 +155,27 @@ sub fill_in_registration_data {
                 }
             }
             send_key $cmd{next};    # all addons selected
+            my @addons_without_license = qw(ha geo we live rt idu ids lgm wsm hpcm);
```

Comment:
Should the name be `@addons_with_license`?

Reply:
That's true, let me fix that. Thanks!